Zapier (#1780)
enkidevs/curriculum: zapier/introduction/zaps-ii/zapier-search-and-create.md

---
author: kapnobatai136

category: must-know

aspects:
  - introduction

type: normal

---

# Search and Create

---
## Content

> Depending on the app that you are using, Zapier allows you to create date after doing a search action.

Reviewer: ```suggestion
Depending on the app that you are using, Zapier allows you to create data after doing a search action.
```

^ maybe quickly re-iterate what a seach action is?

> A good workflow example would be to monitor some data, and when a certain condition is met you create a new record/data.

Reviewer: ```suggestion
For example, a workflow which uses this feature could be monitoring some data change and, when a certain condition is met, it'll create a new data record
```

Let's take another look at the zap we have created. After you receive a message that a person was given access to your base, you want to write a new cell that specifies the number of access days.

> Go ahead and add another Google Sheets step to our zap, right at the end of the flow. This time, we want to create a new column, so you would have to search the `'Choose Action Event'` drop-down for the `'Create Spreadsheet Column'` option.

Reviewer: so this will only create the column but not fill in values. the latter will be discussed in a later insight, right? if so, this is worth mentioning to avoid confusion

![search-and-create](https://img.enkipro.com/4da04371ac9c9efc7769f35c39b275b2.png)

---

> ## Practice

Reviewer: one question that came to mind could be formulated like this:

What actions are available for a Google Sheet step in a zapier workflow? If unsure, you can always check the app page -> link to page
